### PR TITLE
Remove logging from azblob reader

### PR DESCRIPTION
**Pull request by honourfish**
also move accountName into options parameter

re: [AB#9491](https://dev.azure.com/jitsuin/0629f48c-3979-4bbc-9026-cb06b3dfd0ae/_workitems/edit/9491)

### CODE REVIEW

File: azblob/reader.go [hunk 1]
```diff
@@ -67,23 +65,16 @@ func NewReaderNoAuth(accountName string, url string, container string) (Reader,
 		nil,
 	)
 	if err != nil {
-		logger.Sugar.Infof("unable to create serviceclient %s: %v", url, err)
 		return nil, err
```
Comment (eccles) [inline azblob/reader.go:68]: For future work: propagating an error from  a third party service like this is not considered good practice. Usually one defines an error type, wraps the error with '%v' and returns that.

This is just a comment...

Comment (honourfish) [reply azblob/reader.go:68]: agreed! 

We should have a TD ticket to address this, especially in our common go repo :)

**Review by robinbryce — APPROVED** [review]
LGTM. Thankyou very much for sorting out the mess I left there!

**Comment by honourfish** [comment]
> LGTM. Thankyou very much for sorting out the mess I left there!

np :) no mess left!

**Pull request merged**